lib/rucio/daemons/reaper/reaper.py

```diff
@@ -690,7 +690,6 @@ def run(
 
     :param threads:                The total number of workers.
     :param chunk_size:             The size of chunk for deletion.
-    :param threads_per_worker:     Total number of threads created by each worker.
```

Comment:
Please specify in the commit message that this affects only the docstring.

Reply:
Addressed in https://github.com/rucio/rucio/pull/7472/commits/2b9210c2523745aa8fc737be205ecdb8e768274a